starwars/src/components/List.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import StarWars from './StarWars'
+
+const List = props => {
+    return(
+        <div className='list'>
+            {props.charList.map(charList => <StarWars charList={charList} key={charList.created}/>)}
```

Comment:
```suggestion
            {props.charList.map(char => <StarWars char={char} key={charList.created}/>)}
```
Yeah, so this is my suggestion because it's more semantic. If you think about it, the individual items that you're mapping over are individual character objects.